app/js/components/common/Header.jsx

```diff
@@ -55,15 +55,15 @@ export default class Header extends Component {
     this.fetchLocation('/session').then((response) => {
       this.setState((prevState, props) => {
         return {
-          currentUser: response.user.display
+          currentUser: response.user ? response.user.display : ''
         };
       });
     });
 
     this.fetchCurrentUserLocation('/appui/session').then((response) => {
       this.setState((prevState, props) => {
         return {
-          currentLocationTag: response.sessionLocation.display,
+          currentLocationTag: response.sessionLocation ? response.sessionLocation.display : ''
```

Comment:
what does this change do?

Reply:
It first ascertains that there is a sessionLocation before trying to access its display property